doc/devdocs/modules/FileExplorer/monaco/readme.md

```diff
@@ -0,0 +1,65 @@
+# Developer Preview (Monaco)
+
+Developer preview is based on [Microsoft's Monaco Editor](https://microsoft.github.io/monaco-editor/) which is maintained by the Visual Studio Code team.
+
+## Update Monaco Editor
+
+1. Download Monaco editor with [npm](https://www.npmjs.com/): `npm i monaco-editor`.
+2. Delete everything except the `min` folder (the minimised code) from the downloaded files.
+3. Copy the `min` folder into the `/src/modules/previewpane/MonacoPreviewHandler/monacoSRC` folder of the PowerToys project.
+4. Generate the JSON file as described in the [Generate monaco_languages.json file](#generate-monaco_languagesjson-file) section.
+
+## Add a new language definition
+
+As an example on how to add a new language definition you can look at the [Pull Request that adds the REG language definition](https://github.com/microsoft/PowerToys/pull/17183).
```

Comment:
this is a bit unclear looking at that PR what is needed.  it is touching files that i don't think are directly needed.  feels like more was done in that PR.